autopts/wid/tmap.py

```diff
@@ -127,6 +127,8 @@ def hdl_wid_114(params: WIDParams):
     """Please advertise with Broadcast Audio Announcement (0x1852) service data"""
 
     # advertisement started in hdl_wid_506
+    if params.test_case_name == 'TMAP/BMS/CTXT/BV-01-C':
+        hdl_wid_506(params)
```

Comment:
Is this not called by PTS?

Reply:
No, I couldn't get it to call hdl_wid_506. I was expecting it to be as part of the advertising test case.

Reply:
Should this be a precondition for the test instead then?

Reply:
Yes but not sure what condition to set